website/docs/build-apps/how-to-guides/import-export-app-entities.md

```diff
@@ -0,0 +1,33 @@
+---
+description: A guide that outlines steps on how to export entities from a page to a JSON file and subsequently import it into another page.
+---
+
+# Reuse App Entities
+
+When working on complex Appsmith applications, you might find the need to reuse a particular set of widgets, queries, JSObjects, and more across pages. Instead of recreating these entities from scratch each time, Appsmith provides a way for you to export entities from one page and import them into another. This guide outlines steps on how to export entities from a page to a JSON file and subsequently import it into another page.
+
+## Export entities
+
+Follow these steps to export your desired entities:
+
+* Open the Appsmith editor and access the app containing the page you wish to export.
+* Click the three vertical dots next to the page name for more actions, and select the **Export** option from the dropdown menu. A dialog box will appear, allowing you to choose the entities you want to reuse.
+<figure>
```

Comment:
remove this, add an arcade at the top

Reply:
I don't think arcade is needed here, as we are just showing them clicks on entities that don't add any value to the user. A screenshot should be okay here.